tests/unit/conf/Config.js

```diff
@@ -89,3 +90,59 @@ describe('Site name', () => {
         assert.deepStrictEqual(newConfig.bootstrapList, expectedBootstrapList);
     });
 });
+
+describe('Config', () => {
+    describe('getReplicationSiteDestConfig', () => {
+        afterEach(() => {
+            delete process.env.BACKBEAT_CONFIG_FILE;
+            sinon.restore();
+        });
+        it('should return replication site destination config', () => {
+            process.env.BACKBEAT_CONFIG_FILE = `${__dirname}/configs/replicationMultiDestConfig.json`;
+            const conf = new Config();
+            const destConfig = conf.getReplicationSiteDestConfig('aws3');
+            assert.deepStrictEqual(destConfig, {
+                transport: 'https',
+                auth: {
+                    type: 'service',
+                    account: 'service-replication-3',
+                },
+                bootstrapList: [
```

Comment:
it may/could happen that `BACKBEAT_CONFIG_FILE` is set before the tests begin (and some test may depend on it...) : the really safe way to handle this is to store the previous value of BACKBEAT_CONFIG_FILE before the test (beforeEach), so that eventually we can either delete the env variable if it was not set or reset it to its original value otherwise